src/client/components/Button/ButtonComponent.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ButtonComponent.styles.css';
+
+export default function ButtonComponent({ title, backgroundColor, onClick }) {
+  return (
+    <div className="button-component">
+      <button
+        type="button"
+        onClick={onClick}
+        className="button button--primary button--medium"
+        style={{ backgroundColor }}
+      >
+        {title}
+      </button>
+    </div>
+  );
+}
+
+ButtonComponent.propTypes = {
+  title: PropTypes.string.isRequired,
+  onClick: PropTypes.func,
```

Comment:
You are specifying here that the onClick will receive a function, so probably the default prop for it should also be a function :)

Reply:
okay .. i added function